include/contractor/contractor_config.hpp

```diff
@@ -73,6 +73,7 @@ struct ContractorConfig
     bool use_cached_priority;
 
     unsigned requested_num_threads;
+    double verify_weights;
```

Comment:
again, this name does not fit the use-case for me. `verify_weights` sounds like a `bool` to me. My suggestion would be to improve the name and add some commentary on how it is used. Also we need to add a reasonable default.